packages/nimble-components/src/select/tests/select.spec.ts

```diff
@@ -42,7 +42,7 @@ async function checkFullyInViewport(element: HTMLElement): Promise<boolean> {
                     resolve(false);
                 }
             },
-            { threshold: 1.0 }
+            { threshold: 1.0, root: document }
```

Comment:
I don't see it in the PR description. Any comment on the Safari support?: https://github.com/ni/nimble/issues/1208#issuecomment-1523867374

Reply:
I hadn't made a point of it since we don't test on Safari, but I've now added a comment to the source, just in case we start at some point.

Reply:
Add a link to the relevant safari testing issue

Reply:
Added